contents/handbook/company/team/tim-glaser.md

```diff
@@ -0,0 +1,64 @@
+
+## Areas of responsibility
+
+- Make sure we move fast
+- Make sure the engineering team has all the context it needs
+- Make sure the engineering team is happy
+- Make sure we're building in the right direction
+- Sounding board to James Hawkins (CEO)
+
+## Quirks
+
+- I come out with opinions quickly and strongly. They are actually weekly held so please push back when you disagree, I'll take it well.
+- I will likely talk way too much. Please just interject/talk over me.
+- A lot of what I say sounds like a definitive statement ("this is what we're going to do") when I actually mean "here's something we could do".
+- I'm easily distracted in meetings and can come across as disinterested. It's something I'm working on but if you notice I'm drifting off please mention it.
+- Weekends are holy to me and I'll almost never work on a weekend (and don't expect anyone else to!). I also don't like to have meetings after ~8pm my time, or before 10-11am.
+- Please don't message me "hey" and then spend 3 minutes typing your question. Adding those two together is fine.
+- I like my meetings clustered together.
+
+## What I value
+
+- People who get things done without me having to chase. You're golden if I mention something off hand and it gets done.
+- Receiving feedback. Please give me a ton of feedback, I still have a huge amount to go and will only improve if I get feedback.
+- When I give feedback, I'd appreciate if you consider it first before defending yourself. I may have gotten it wrong and feel free to push back, but I don't like it when people start defending themselves straight away.
+- I'm not a big fan of meetings. Please write your thoughts up in a GitHub issue/PR first. We can always have a meeting after that which will likely be more productive.
+- I like short, to the point writing. Use short words, bullet points and screenshots.
+- Directness. If you don't like something please just say so.
+
+
+## How I can help you
+
+- I can help you figure out how to build something in 1/2 the time you think it should take
+- I can help you figure out what you should be working on
+- I can help you figure out what to do in your career
+- I can help be a rubber duck
+- I can help bounce ideas around
+
+## How you can help me
+
+- Come to 1:1s with an agenda and clear things I can help with
+- Give me feedback
+- Bring up problems, don't hide them. As a startup we'll always have a million problems, it's our job to surface those and fix the important ones.
+
+## Nomadding
+
+I currently don't have a fixed address, and tend to move places every 2-3 months. If I remember to update it, [you can see where I am and where I'm going here](https://nomadlist.com/@timgl). I'd love to meet up with anyone if it's within a reasonable distance (and sometimes even if it's not). Would love for you to reach out and organise something.
```

Comment:
Minor US English typo "organize".

Reply:
Gives me an idea that I could configure the bot to pick up the most common typos from British English. 

Also I think in a page like this it's fine, because it's a personal thing.

Reply:
haha yeah given it's written from my POV i'd write organise.